src/bindings/python/tests/mock/mock_py_frontend/CMakeLists.txt

```diff
@@ -2,43 +2,46 @@
 
 file(GLOB_RECURSE LIBRARY_SRC ${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp)
 file(GLOB_RECURSE LIBRARY_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/include/*.hpp)
 
 source_group("src" FILES ${LIBRARY_SRC})
 source_group("include" FILES ${LIBRARY_HEADERS})
 
-add_library(${TARGET_FE_NAME} ${LIBRARY_SRC} ${LIBRARY_HEADERS})
+# Build as shared library because of static members.
+# If build as static library then these members will be duplicated e.g. per pybind shared library and test will fail.
+# Using static methods (statistics) from pybind_mock_frontend access different data then
+# mock fornt end created by pyopenvino.
+add_library(${TARGET_NAME} SHARED ${LIBRARY_SRC} ${LIBRARY_HEADERS})
+add_library(openvino::frontend::mock_py ALIAS ${TARGET_NAME})
 
-target_include_directories(${TARGET_FE_NAME}
-        PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include
-        PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src)
+target_include_directories(${TARGET_NAME} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
 
-set(DEPENDENCIES openvino::runtime)
-set(DEFINITIONS IMPLEMENT_OPENVINO_API)
+target_compile_definitions(${TARGET_NAME} PRIVATE
+    IMPLEMENT_OPENVINO_API
+    $<$<BOOL:${ENABLE_OV_ONNX_FRONTEND}>:ENABLE_OV_ONNX_FRONTEND>
+    $<$<BOOL:${ENABLE_OV_TF_FRONTEND}>:ENABLE_OV_TF_FRONTEND>
+    $<$<BOOL:${ENABLE_OV_PADDLE_FRONTEND}>:ENABLE_OV_PADDLE_FRONTEND>)
 
-if (ENABLE_OV_ONNX_FRONTEND)
-    list(APPEND DEPENDENCIES openvino::frontend::onnx)
-    list(APPEND DEFINITIONS ENABLE_OV_ONNX_FRONTEND)
-endif()
-
-if (ENABLE_OV_TF_FRONTEND)
-    list(APPEND DEPENDENCIES openvino::frontend::tensorflow)
-    list(APPEND DEFINITIONS ENABLE_OV_TF_FRONTEND)
-endif()
+target_link_libraries(${TARGET_NAME} PUBLIC
+    $<$<BOOL:${ENABLE_OV_ONNX_FRONTEND}>:openvino::frontend::onnx>
+    $<$<BOOL:${ENABLE_OV_TF_FRONTEND}>:openvino::frontend::tensorflow>
+    $<$<BOOL:${ENABLE_OV_PADDLE_FRONTEND}>:openvino::frontend::paddle>)
 
-if (ENABLE_OV_PADDLE_FRONTEND)
-    list(APPEND DEPENDENCIES openvino::frontend::paddle)
-    list(APPEND DEFINITIONS ENABLE_OV_PADDLE_FRONTEND)
-endif()
+ov_add_library_version(${TARGET_NAME})
+add_clang_format_target(${TARGET_NAME}_clang FOR_TARGETS ${TARGET_NAME})
 
-target_compile_definitions(${TARGET_FE_NAME} PRIVATE ${DEFINITIONS})
-target_link_libraries(${TARGET_FE_NAME} PRIVATE ${DEPENDENCIES})
-
-add_clang_format_target(${TARGET_FE_NAME}_clang FOR_TARGETS ${TARGET_FE_NAME})
-
-install(TARGETS ${TARGET_FE_NAME}
+if(BUILD_SHARED_LIBS)
+    install(TARGETS ${TARGET_NAME}
         RUNTIME DESTINATION ${OV_CPACK_RUNTIMEDIR} COMPONENT tests EXCLUDE_FROM_ALL
         LIBRARY DESTINATION ${OV_CPACK_LIBRARYDIR} COMPONENT tests EXCLUDE_FROM_ALL)
+else()
+    set(pyversion python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})
+    install(TARGETS ${TARGET_NAME}
+        RUNTIME DESTINATION tests COMPONENT tests EXCLUDE_FROM_ALL
+        LIBRARY DESTINATION tests COMPONENT tests EXCLUDE_FROM_ALL
```

Comment:
still suppose that we need to install such test frontends unconditionally on `tests` folder and use `OV_FRONTEND_PATH`

Reply:
Sure can do that but need help what exactly where set this OV_DRONTEND_PATH.
Still got issue with Azure CI Paths set but test skipped (because cannot be imported by python) it did not reach to FEM to load plugin). Need some help how it should be set.
I can manage that works local machine in both cases static and shared build but not on CI.